Submissions/Michelleivan.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Michelle Ivan",
+    "img": " file:///C:/Users/itsmi/Downloads/1A18F5EA-996C-40B2-8107-03D014C5E390.webp",
```

Comment:
This won't work, the image is on your local computer, but not available to everyone else. If you do not want to use your github avatar, you need to upload the picture somewhere on the internet.